biocypher/_ontology.py

```diff
@@ -514,15 +566,37 @@ def _assert_join_node(self, adapter: OntologyAdapter) -> None:
                 join node in the head ontology.
         """
 
-        head_join_node = adapter.get_head_join_node()
+        head_join_node = None
+        head_join_node_label = adapter.get_head_join_node().replace("_", " ")
```

Comment:
In other methods, we make replacing underscores contingent on the `rename_nodes` parameter; why do we not do that here? We also convert to lowercase in the other places when we rename, why not here?